### PR TITLE
Fix for usage from China

### LINKED ISSUE #493: Usage from China requires different API URL
If using Google Maps APIs from China a different URL should be used, see https://developers.google.com/maps/faq#china_ws_access.

https://maps.googleapis.com should be replaced with http://maps.google.cn, since the regular one is not accessible from within China.

### PR DESCRIPTION
**Pull request by emilpalsson**
Use China specific API URL if setting bootstrapURLKeys.region to 'cn'.

Fixes #493

### CODE REVIEW

**Comment by itsmichaeldiego** [comment]
@emilpalsson Thanks for this, it is looking good, would you mind adding this to the docs? As soon as you do that I will release a new version fixing this.

File: src/utils/loaders/google_map_loader.js [hunk 1]
```diff
@@ -62,8 +62,12 @@ export default function googleMapLoader(bootstrapURLKeys) {
       ''
     );
 
+    const url = bootstrapURLKeys.region === 'cn'
```
Comment (jooj123) [inline src/utils/loaders/google_map_loader.js]: what about other regions ?

Comment (itsmichaeldiego) [reply src/utils/loaders/google_map_loader.js]: @jooj123 I believe the only region that has a problem with this is China. But we could confirm

Comment (emilpalsson) [reply src/utils/loaders/google_map_loader.js]: Exactly, China is the only region mentioned on the Google Maps FAQ (see link in the issue), and the only region I've experienced problems with.

**Comment by emilpalsson** [comment]
@itsmichaeldiego Great! Of course, I'll see if I can find a good spot for this in the docs right now.

**Comment by emilpalsson** [comment]
@itsmichaeldiego I've added this to the docs, let me know if you want me to structure it differently. I also added a toLocaleLowerCase() for case insensitivity.

**Comment by romanenko** [comment]
Looks like that if region is not set — it leads to uncaught Error now, with 0.28.0 release

image

I'm setting up my GoogleMap component like this:

```js
      <GoogleMap
        bootstrapURLKeys={{
          key: GOOGLE_MAPS_API_KEY,
          language: intl.locale,
        }}
//...
```

**Comment by emilpalsson** [comment]
@romanenko Yes, I'm sorry for this. There is already a new PR to fix this (#494).

**Pull request merged**